### ISSUE TITLE
Dataset and parameters

### ISSUE THREAD

**Issue by Di40** [pschaus/csrnet_pip#1]
Can you please share which dataset, and which parameters (optimizer, learning rate, loss, epochs, etc.) are used for training the model? Thank you in advance!
I am hoping to reuse the weights and fine-tune them on the Mall dataset. Without fine-tunning, the results are still decent!
